core/foundation/inc/ROOT/RLogger.hxx

```diff
@@ -176,13 +326,43 @@ public:
 #define R__LOG_PRETTY_FUNCTION __PRETTY_FUNCTION__
 #endif
 
-#define R__LOG_HERE(LEVEL, GROUP) \
-   ROOT::Experimental::RLogEntry(LEVEL, GROUP).SetFile(__FILE__).SetLine(__LINE__).SetFunction(R__LOG_PRETTY_FUNCTION)
-
-#define R__FATAL_HERE(GROUP) R__LOG_HERE(ROOT::Experimental::ELogLevel::kFatal, GROUP)
-#define R__ERROR_HERE(GROUP) R__LOG_HERE(ROOT::Experimental::ELogLevel::kError, GROUP)
-#define R__WARNING_HERE(GROUP) R__LOG_HERE(ROOT::Experimental::ELogLevel::kWarning, GROUP)
-#define R__INFO_HERE(GROUP) R__LOG_HERE(ROOT::Experimental::ELogLevel::kInfo, GROUP)
-#define R__DEBUG_HERE(GROUP) R__LOG_HERE(ROOT::Experimental::ELogLevel::kDebug, GROUP)
+/*
+ Some implementation details:
+
+ - The conditional `RLogBuilder` use prevents stream operators from being called if
+ verbosity is too low, i.e.:
+ ````
+ RLogScopedVerbosity silence(RLogLevel::kFatal);
+ R__LOG_DEBUG() << WillNotBeCalled();
```

Comment:
This comment/doc might not be matching the code anymore.

Reply:
Thanks, fixed in 5528d31fbf1a004a580dbe56d85e7a6d077744d7